maintainers/scripts/pluginupdate.py

```diff
@@ -772,7 +875,7 @@ def rewrite_input(
         fieldnames = ["repo", "branch", "alias"]
         writer = csv.DictWriter(f, fieldnames, dialect="unix", quoting=csv.QUOTE_NONE)
         writer.writeheader()
-        for plugin in sorted(plugins):
+        for plugin in sorted(plugins, key=lambda x: x.name):
```

Comment:
interesting fact: plugin's URI is not unique, but name is. It is because there can be multiple plugins in one repo but on different branches (e.g. different versions, see [harpoon](https://github.com/NixOS/nixpkgs/blob/9caabb561519db70906c1435a3842417f519f1d9/pkgs/applications/editors/vim/plugins/vim-plugin-names#L386-L387)) and name is used as a key in the final result